angular/index.ts

```diff
@@ -43,6 +43,10 @@ export class SelectedIndexValueAccessor extends BaseValueAccessor<SelectableView
     public onTouched = () => { };
 
     public writeValue(value: any): void {
+        if (value === undefined || value === null || value === "") {
+            return null;
```

Comment:
Shouldn't this and the line right after the `if` be changed to:
```ts
if (value === undefined || value === null ||  value === "") {
   this._normalizedValue = null;
}
else {
  this._normalizedValue = convertToInt(value);
}
```
And then leave the rest of logic to execute even if the value is null or empty. This way if the form value is reset to null or empty the `selectedIndex` property of the drop down will be set to null and thus show the hint?